main.js

```diff
@@ -0,0 +1,138 @@
+{
+  const API_URL = 'https://opentdb.com/api.php?amount=10&type=multiple';
+
+  class Quiz {
+    constructor(quizData) {
+      this._quizzes = quizData.results;
+      this._correctAnswersNum = 0;
+    }
+
+    getQuizCategory(index) {
+      return this._quizzes[index - 1].category;
+    }
+
+    getQuizDifficulty(index) {
+      return this._quizzes[index - 1].difficulty;
+    }
+
+    getNumOfQuiz() {
+      return this._quizzes.length;
+    }
+
+    getQuizQuestion(index) {
+      return this._quizzes[index - 1].question;
+    }
+
+    getCorrectAnswer(index) {
+      return this._quizzes[index - 1].correct_answer;
+    }
+
+    getIncorrectAnswers(index) {
+      return this._quizzes[index - 1].incorrect_answers;
+    }
+
+    countCorrectAnswersNum(index, answer) {
+      const correctAnswer = this._quizzes[index - 1].correct_answer;
+      if (answer === correctAnswer) {
+        return this._correctAnswersNum++;
+      }
+    }
+
+    getCorrectAnswersNum() {
+      return this._correctAnswersNum;
+    }
+  }
+
+  const titleElement = document.getElementById('title');
+  const categoryElement = document.getElementById('category');
+  const quizElement = document.getElementById('quiz');
+  const startButton = document.getElementById('btn');
+  const ansContainer = document.getElementById('ans');
+  const difficultyElement = document.getElementById('difficulty');
+
+  startButton.addEventListener('click', () => {
+    startButton.hidden = true;
+    fetchQuizData(1);
```

Comment:
`fetchQuizData` は `async` な関数なので `await` があるべきです。
```suggestion
    await fetchQuizData(1);
```